hs/src/Reach/Simulator/Server.hs

```diff
@@ -626,6 +711,11 @@ app p srcTxt = do
     cat <- webM $ catGraph
     json cat
 
+  get "/dotgraph" $ do
```

Comment:
I think this name is confusing. Dot is a programming language / format. The thing is a graph of global state diagram